resources/js/features/games/components/TopAchieversMainRoot/TopAchieversList/TopAchieversList.tsx

```diff
@@ -0,0 +1,85 @@
+import type { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import {
+  BaseTable,
+  BaseTableBody,
+  BaseTableCell,
+  BaseTableHead,
+  BaseTableHeader,
+  BaseTableRow,
+} from '@/common/components/+vendor/BaseTable';
+import { PlayerBadgeIndicator } from '@/common/components/PlayerBadgeIndicator';
+import { PlayerBadgeLabel } from '@/common/components/PlayerBadgeLabel';
+import { UserAvatar } from '@/common/components/UserAvatar';
+import { usePageProps } from '@/common/hooks/usePageProps';
+import { AwardType } from '@/common/utils/generatedAppConstants';
+import { formatDate } from '@/common/utils/l10n/formatDate';
+import { cn } from '@/utils/cn';
+
+export const TopAchieversListContainerTestId = 'top-achievers-list';
+
+export const TopAchieversList: FC = () => {
+  const { paginatedUsers } = usePageProps<App.Platform.Data.GameTopAchieversPageProps>();
+
+  const { t } = useTranslation();
+
+  if (!paginatedUsers.items.length) {
+    return null;
+  }
+
+  return (
+    <BaseTable
+      containerClassName={cn(
+        'overflow-auto rounded-md border border-neutral-700/80 bg-embed',
+        'light:border-neutral-300 lg:overflow-visible lg:rounded-sm',
+        '',
+        '',
+      )}
+    >
+      <BaseTableHeader>
+        <BaseTableRow className="do-not-highlight">
+          <BaseTableHead>{t('Rank')}</BaseTableHead>
+          <BaseTableHead>{t('User')}</BaseTableHead>
+          <BaseTableHead>{t('Progress')}</BaseTableHead>
+        </BaseTableRow>
+      </BaseTableHeader>
+
+      <BaseTableBody>
+        {paginatedUsers.items.map((achiever) => (
+          <BaseTableRow key={achiever.user.displayName}>
+            <BaseTableCell>{achiever.rank}</BaseTableCell>
+
+            <BaseTableCell>
+              <UserAvatar {...achiever.user} size={32} />
+            </BaseTableCell>
```

Comment:
There's a curious CSS bug here:

![Screenshot 2024-11-23 at 6 38 03 PM](https://github.com/user-attachments/assets/ac25c927-cf61-4eb7-a9e3-871c00f48cea)

This can be resolved by:
```tsx
<BaseTableCell>
  <div className="max-w-fit">
    <UserAvatar {...achiever.user} size={32} />
  </div>
</BaseTableCell>
```

Reply:
Not sure what the bug is, but I made the suggested change.

Reply:
The bug was no matter how far across the table cell I was hovering, the user tooltip card was still appearing.